test/integration/document/PresentationViewer.e2e.test.js

```diff
@@ -25,4 +27,17 @@ describe('Presentation Viewer', () => {
 
     });
 
+    it('Should navigate to a specified page when an internal link is clicked', () => {
+        cy.showPreview(token, fileWithLinksId);
+
+        // Assert document content is present and clickable
+        cy.contains('LINK TO PAGE 1')
+        
+        // Internal links should be rendered with <a> tags by PDF.js
+        cy.get('.bp-content a').click();
```

Comment:
Would it make sense to do a `getByTestId('bp-content').find('a')` instead since we have been trying to avoid querying by css classes?